provider/marathon/marathon_test.go

```diff
@@ -1801,3 +1807,33 @@ func TestParseIndex(t *testing.T) {
 		})
 	}
 }
+
+func TestMarathonGetBasicAuth(t *testing.T) {
+	provider := &Provider{}
+
+	applications := []struct {
```

Comment:
I would argue to keep the the style the same an in a different PR correct them all at once. Would be confusing to any one else giving a contribution why there are two styles